readthedocs/doc_builder/python_environments.py

```diff
@@ -129,13 +135,14 @@ def install_core_requirements(self):
         if self.project.documentation_type == 'mkdocs':
             requirements.append('mkdocs==0.15.0')
         else:
-            if self.project.has_feature(Feature.USE_SPHINX_LATEST):
-                # We will assume semver here and only automate up to the next
-                # backward incompatible release: 2.x
-                requirements.append('sphinx<2')
-            else:
-                requirements.append('sphinx==1.5.6')
+            # We will assume semver here and only automate up to the next
+            # backward incompatible release: 2.x
             requirements.extend([
+                self.project.get_feature_value(
+                    Feature.USE_SPHINX_LATEST,
+                    'sphinx<2',
+                    'sphinx==1.5.6',
```

Comment:
I like this pattern, more explicit. Could maybe use kwargs here, to make it more obvious what the options are.